<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>memur_islem</class>
 <widget class="QWidget" name="memur_islem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>401</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title">
    <string>ÖĞRENCİ KAYIT</string>
   </property>
   <widget class="QLineEdit" name="ogr_ad">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:2px;
border-style:solid;
border-color: rgb(230, 115, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ogr_soyad">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:2px;
border-style:solid;
border-color: rgb(230, 115, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ogr_numara">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:2px;
border-style:solid;
border-color: rgb(230, 115, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ogr_e_posta">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:2px;
border-style:solid;
border-color: rgb(230, 115, 0);</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="ogr_sifre">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:2px;
border-style:solid;
border-color: rgb(230, 115, 0);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ogr_bolum">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Bilişim_Sis_Müh</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Biyomedikal_Müh</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Otomotiv_Müh</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="ogr__kayit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color: rgb(230, 115, 0);</string>
    </property>
    <property name="text">
     <string>KAYDET</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>AD:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SOYAD:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>NUMRARA:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>E-POSTA:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>BÖLÜM:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ŞİFRE:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Dönem:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Kayıt Tarihi:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>50</y>
     <width>391</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title">
    <string>EĞİTİM GÖREVLİSİ  DERS AÇMA</string>
   </property>
   <widget class="QLineEdit" name="egitim_gor_ad">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:2px;
border-style:solid;
border-color: rgb(230, 115, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="egitim_gor_soyad">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:2px;
border-style:solid;
border-color: rgb(230, 115, 0);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ders_acma_bolum">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Bilişim_Sis_Müh</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Biyomedikal_Müh</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Otomotiv_Müh</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="ders_kaydet">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color: rgb(230, 115, 0);</string>
    </property>
    <property name="text">
     <string>KAYDET</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>AD:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SOYAD:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bölüm:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="akademisyen_ders">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ders:</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="arka_plan">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>831</width>
     <height>481</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>arka_plan</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
